README.md

```diff
@@ -36,7 +36,8 @@ processed.
 `Future` objects encapsulate the result of an async procedure, upon successful
 completion, and a list of callbacks to be scheduled after completion or
 cancellation. (These explicit callbacks are rarely used outside Chronos, being
```

Comment:
callbacks are scheduled when Future changes it state from `FutureState.Pending`. So callbacks are scheduled also when Future got failed or cancelled.

Reply:
> A Promise/Future is a proxy for a value not necessarily known when the promise/Future is created. It allows you to associate handlers with an asynchronous action's eventual success value or failure reason. This lets asynchronous methods return values like synchronous methods: instead of immediately returning the final value, the asynchronous method returns a promise to supply the value at some point in the future.

Reply:
Clarified.